todo_app/app.py

```diff
@@ -1,11 +1,24 @@
-from flask import Flask
-
+from flask import Flask, render_template, request
+from todo_app.data import session_items
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session_items.get_items()
+    return render_template('index.html', items=items)
+
+@app.route('/addTask', methods=['POST'])
+def addTask():
+    newTask = request.form['TaskName']
```

Comment:
It would be nice to be consistent with your naming style. The convention in Python is to use snake_case for variable names.

i.e. `new_task` rather than `newTask`.

It just makes it easier to read/write/search

Reply:
changing now